dbschema/models.py

```diff
@@ -48,6 +49,17 @@ class DialogQuestions(Base):
     __tablename__ = 'dialog_questions'
     question_id = Column(Integer, primary_key=True)
     question_description = Column(String)
+    
+    
+class FirstAidKit(Base):
```

Comment:
Maybe add a docstring, so people new to the project now what the *#@%$& we mean with `FirstAidKit` 😉